### PR TITLE
{lib}[foss/2022a] tensorboard v2.10.0 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by surak**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/t/tensorboard/tensorboard-2.10.0-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    }),
+    ('google-auth-oauthlib', '0.4.6', {
+        'checksums': ['a90a072f6993f2c327067bf65270046384cda5a8ecb20b94ea9a687f1f233a7a'],
+    }),
+    ('grpcio', '1.49.1', {
+        'modulename': 'grpc',
+        'preinstallopts': "export GRPC_PYTHON_BUILD_EXT_COMPILER_JOBS=%(parallel)s && ",
+        'checksums': ['d4725fc9ec8e8822906ae26bb26f5546891aa7fbc3443de970cc556d43a5c99f'],
+    }),
+    ('importlib_metadata', '4.12.0', {
```
Comment (smoors) [inline easybuild/easyconfigs/t/tensorboard/tensorboard-2.10.0-foss-2022a.eb:50]: @surak 
`importlib_metadata` (version 4.11.3) is already included in Python 3.10.4 
is that not recent enough?

Comment (surak) [reply easybuild/easyconfigs/t/tensorboard/tensorboard-2.10.0-foss-2022a.eb:50]: Yes, it is. It seems that eb can't work with --update-pr if you used --read-only-installdir before, but you are correct and it works just fine

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @surak!

**Pull request merged**